### PR TITLE
feat: Support new models (XR, XS, XS Max) and iPad 6th Gen

### LINKED ISSUE #493: app support for new iPhone devices(iPhone XR, iPhone XS, iPhone XS MAX)
iPhone new device models does not exist in deviceNamesByCode list (iPhone XR, iPhone XS, iPhone XS MAX)

### PR DESCRIPTION
**Pull request by duro**
Fixes #493 and replaces PR #495

### CODE REVIEW

File: ios/RNDeviceInfo/RNDeviceInfo.m [hunk 1]
```diff
@@ -103,10 +103,17 @@ - (NSString*) deviceName
                               @"iPhone9,4" :@"iPhone 7 Plus",   // (model A1784 | Global)
                               @"iPhone10,3":@"iPhone X",        // (model A1865, A1902)
                               @"iPhone10,6":@"iPhone X",        // (model A1901)
+                              @"iPhone11,8":@"iPhone XR",       // (model A1882, A1719)
```
Comment (mattlubner) [inline ios/RNDeviceInfo/RNDeviceInfo.m]: Is it alright for there to be duplicate entries?

Comment (mattlubner) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: Ah, I should've said "duplicate keys", my bad. Line 106 has an entry for `@"iPhone11,8"` and line 116 has a duplicate entry for the same key.

Comment (duro) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: @mattlubner I see what you are saying. Likely somethign weird with the merge. I'll clean that up

File: ios/RNDeviceInfo/RNDeviceInfo.m [hunk 2]
```diff
@@ -103,10 +103,16 @@ - (NSString*) deviceName
                               @"iPhone9,4" :@"iPhone 7 Plus",   // (model A1784 | Global)
                               @"iPhone10,3":@"iPhone X",        // (model A1865, A1902)
                               @"iPhone10,6":@"iPhone X",        // (model A1901)
+                              @"iPhone11,2":@"iPhone XS",       //
```
Comment (mattlubner) [inline ios/RNDeviceInfo/RNDeviceInfo.m]: Ahh sorry @duro! I thought you would've seen these other duplicate keys too. One for `@"iPhone11,2"` (lines 106 and 112) and another for `@"iPhone11,4"` (lines 107 and 113). Those are all of them, promise! 🤦‍♂️

Comment (duro) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: Cleaned those out

**Comment by mattlubner** [comment]
Sorry for being the squeaky wheel on this one, y'all! @machour are you the right person to get this PR merged? If not, could you point us in the right direction?

**Comment by machour** [comment]
Just released 0.22.6, enjoy :)

**Pull request merged**